[2021-10-29 11:52:42,080] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: transformation.append_value_sharepoint manual__2021-10-29T06:22:13.648343+00:00 [queued]>
[2021-10-29 11:52:42,097] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: transformation.append_value_sharepoint manual__2021-10-29T06:22:13.648343+00:00 [queued]>
[2021-10-29 11:52:42,098] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-10-29 11:52:42,098] {taskinstance.py:1242} INFO - Starting attempt 1 of 2
[2021-10-29 11:52:42,098] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-10-29 11:52:42,120] {taskinstance.py:1262} INFO - Executing <Task(PythonOperator): append_value_sharepoint> on 2021-10-29 06:22:13.648343+00:00
[2021-10-29 11:52:42,127] {standard_task_runner.py:52} INFO - Started process 9602 to run task
[2021-10-29 11:52:42,140] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'transformation', 'append_value_sharepoint', 'manual__2021-10-29T06:22:13.648343+00:00', '--job-id', '29', '--raw', '--subdir', 'DAGS_FOLDER/transformation.py', '--cfg-path', '/tmp/tmpfhh7g7x4', '--error-file', '/tmp/tmppufh8u9b']
[2021-10-29 11:52:42,151] {standard_task_runner.py:77} INFO - Job 29: Subtask append_value_sharepoint
[2021-10-29 11:52:42,218] {logging_mixin.py:109} INFO - Running <TaskInstance: transformation.append_value_sharepoint manual__2021-10-29T06:22:13.648343+00:00 [running]> on host user-Latitude-3410
[2021-10-29 11:52:42,267] {taskinstance.py:1414} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=shindesushant818@gmail.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=transformation
AIRFLOW_CTX_TASK_ID=append_value_sharepoint
AIRFLOW_CTX_EXECUTION_DATE=2021-10-29T06:22:13.648343+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-10-29T06:22:13.648343+00:00
[2021-10-29 11:52:46,850] {logging_mixin.py:109} INFO - Shareplum HTTP Post Failed : 423 Client Error: Locked for url: https://orienttechnologies.sharepoint.com/sites/sushant_ETL/_api/web/GetFileByServerRelativeUrl('/sites/sushant_ETL/Shared%20Documents/tranformations/tranformation.xlsx') error of line number 119
[2021-10-29 11:52:46,851] {python.py:152} INFO - Done. Returned value was: None
[2021-10-29 11:52:46,870] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=transformation, task_id=append_value_sharepoint, execution_date=20211029T062213, start_date=20211029T062242, end_date=20211029T062246
[2021-10-29 11:52:46,934] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-10-29 11:52:46,983] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
